Consul Secure ACL Vault scenario (#355)
hashicorp/katakoda: consul-secure-acl-vault-hashiconf/step1.md

> The Consul secrets engine generates Consul API tokens dynamically based on 
> Consul ACL policies, and must be enabled before it can perform its functions.

Reviewer: ```suggestion
The Consul secrets engine generates Consul API tokens in Vault dynamically based on 
Consul ACL policies, and must be enabled before it can perform its functions.
```
Clarifies what process is generating the Vault tokens.. I could be wrong here.. maybe it should be "Vault's Consul secrets engine.." instead

`vault secrets enable consul`{{execute T1}}

Example output:

```
Success! Enabled the consul secrets engine at: consul/
```

### Configure Vault to connect and authenticate to Consul

The secrets engine requires a management token, with unrestricted privileges, to interact with Consul.

First, create a management token in Consul:

`consul acl token create \
    -policy-name=global-management \
    -format=json | tee ./assets/secrets/acl_management.json`{{execute T1}}

Export the management token as an environment variable:

`export CONSUL_MGMT_TOKEN=$(cat ./assets/secrets/acl_management.json | jq -r ".SecretID")`{{execute T1}}

> Once you have the Consul secret backend enabled,
> configure access with Consul's address and management token:

Reviewer: ```suggestion
Configure Vault's Consul secret backend with Consul's address and management token:
```
Changed to succinctly describe what is happening with the code below.
